test/rcedit-test.js

```diff
@@ -107,6 +108,24 @@ describe('rcedit(exePath, options, callback)', function () {
     })
   })
 
+  it('supports setting requestedExecutionLevel to requireAdministrator', function (done) {
+    var options = {
+      'requested-execution-level': 'requireAdministrator'
+    }
+
+    rcedit(exePath, options, function (error) {
```

Comment:
Before you run rcedit, you should assert that the same grep that's run after running `rcedit` fails.